js/whitebit.js

```diff
@@ -503,42 +572,65 @@ module.exports = class whitebit extends Exchange {
         //         "isBuyerMaker":false
         //     }
         //
+        // orderTrades (v4Private)
+        //       {
+        //          "time": 1593342324.613711,      // Timestamp of executed order
+        //          "fee": "0.00000419198",         // fee that you pay
+        //          "price": "0.00000701",          // price
+        //          "amount": "598",                // amount in stock
+        //          "id": 149156519,                // trade id
+        //          "dealOrderId": 3134995325,      // completed order Id
+        //          "clientOrderId": "customId11",  // custom order id; "clientOrderId": "" - if not specified.
+        //          "role": 2,                      // Role - 1 - maker, 2 - taker
+        //          "deal": "0.00419198"            // amount in money
+        //       }
         let timestamp = this.safeValue (trade, 'time');
         if (typeof timestamp === 'string') {
             timestamp = this.parse8601 (timestamp);
         } else {
             timestamp = parseInt (timestamp * 1000);
         }
+        const orderId = this.safeString (trade, 'dealOrderId');
         const priceString = this.safeString (trade, 'price');
-        const amountString = this.safeString2 (trade, 'amount', 'volume');
+        let amountString = this.safeString (trade, 'deal');
+        if (amountString === undefined) {
+            amountString = this.safeString2 (trade, 'amount', 'volume');
+        }
         const cost = this.parseNumber (Precise.stringMul (priceString, amountString));
         const price = this.parseNumber (priceString);
         const amount = this.parseNumber (amountString);
         const id = this.safeString2 (trade, 'id', 'tradeId');
         let side = this.safeString (trade, 'type');
         if (side === undefined) {
             const isBuyerMaker = this.safeValue (trade, 'isBuyerMaker');
-            side = isBuyerMaker ? 'buy' : 'sell';
+            if (side !== undefined) {
+                side = isBuyerMaker ? 'buy' : 'sell';
+            }
         }
         let symbol = undefined;
         if (market !== undefined) {
             symbol = market['symbol'];
```

Comment:
Let's use `safeSymbol` here (see usage in other `parseTrade` implementations in the existing exchanges).